.github/ISSUE_TEMPLATE/Performance.md

```diff
@@ -24,17 +24,33 @@ Please list out the steps you think we should take to solve this issue.
 Note: These should be the same as the benchmarks collected before any changes.
 
 ## Platforms:
-<!---
-Check off any platforms that are affected by this issue
---->
-Which of our officially supported platforms is this issue occurring on?
-- [ ] Android: Native
+Select the officially supported platforms where the issue was reproduced:
+- [ ] Android: Standalone
+- [ ] Android: HybridApp
 - [ ] Android: mWeb Chrome
-- [ ] iOS: Native
+- [ ] iOS: Standalone
+- [ ] iOS: HybridApp
 - [ ] iOS: mWeb Safari
+- [ ] Windows: Chrome
```

Comment:
We officially support windows? That's news to me

Reply:
Yeah, just not desktop.